cli/cli.ts

```diff
@@ -3906,22 +3910,24 @@ export function staticpkgAsync(parsed: commandParser.ParsedCommand) {
         .then(() => bump ? bumpAsync() : Promise.resolve())
         .then(() => internalBuildTargetAsync({ packaged: true }));
     if (ghpages) return p.then(() => ghpPushAsync(builtPackaged, minify));
-    else return p.then(() => internalStaticPkgAsync(builtPackaged, route, minify));
+    else return p.then(() => internalStaticPkgAsync(builtPackaged, route, minify, disableAppCache));
 }
 
-function internalStaticPkgAsync(builtPackaged: string, label: string, minify: boolean) {
+function internalStaticPkgAsync(builtPackaged: string, label: string, minify: boolean, disableAppCache?: boolean) {
     const pref = path.resolve(builtPackaged);
     const localDir = label == "./" ? "./" : label ? "/" + label + "/" : "/"
     return uploadCoreAsync({
         label: label || "main",
         pkgversion: "0.0.0",
         fileList: pxtFileList("node_modules/pxt-core/")
             .concat(targetFileList())
-            .concat(["targetconfig.json"]),
+            .concat(["targetconfig.json"])
+            .concat(nodeutil.allFiles("built/hexCache")),
```

Comment:
why are we uploading those files? they are already blobs in the cloud

Reply:
This is only for staticPkg, we need the hex cache to be bundled inside the pkg